cleansio/audio/Libs used

```diff
@@ -0,0 +1,8 @@
+convert.py
+    AudioSegment (python)
```

Comment:
Can you create a new page on the Wiki for this?

Reply:
OK

Reply:
Hey, I meant the library list from #17, not the sampling class for the wiki page. Also check out my PR